### ISSUE TITLE
Creating your own spreadsheet name

### ISSUE THREAD

**Issue by xshadow** [audreyt/ethercalc#295]
I would like to suggest an additional option for spreadsheet creation.

It would be nice, if its possible to create a field where the user could enter a spreadsheet name on the ethercalc landing page. Per default this field should be filled with the random alphanumeric characters, that are used at the moment for spreadsheet creation. 

That would give the user the feeling of being in control and a better memorable spreadsheet name.

Here is an example: https://pad.systemli.org/

**Comment by audreyt** [comment]
That makes sense. Would you like to modify `start.html` so it works like you suggested? No backend coding required.

**Comment by mathieu-aubin** [comment]
So is this still  open? If i get some time soon i'll work on that. @audreyt could you please confirm this is still up. Also any rules/styles for the input you'd like to stick to? Thanks i appreciate.

**Comment by audreyt** [comment]
This is still up. There is no style guideline so just make up something !
Thanks for looking into it.

Mathieu Aubin <notifications@github.com> 於 2016年5月31日 週二 14:11 寫道：

> So is this still open? If i get some time soon i'll work on that. @audreyt
> <https://github.com/audreyt> could you please confirm this is still up.
> Also any rules/styles for the input you'd like to stick to? Thanks i
> appreciate.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/audreyt/ethercalc/issues/295#issuecomment-222600574>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABQ86wLJBG84MHUp2SYwYSwKM7GThqlks5qG9EjgaJpZM4G-xln>
> .
>
-- 

Cheers,
Audrey

**Comment by eddyparkinson** [comment]
@mathieu-aubin    You mean like this:
framacalc.org

No idea who created it - but would be good to have something like it in ethercalc.
@PhilLAL looks to know about it.

**Comment by mathieu-aubin** [comment]
Yeah, actually was just asking if it was still open -- it's on my todo list when i get a few minutes for my own fun projects like this. Hehh, thanks for the input tho.

**Comment by PhilLAL** [comment]
framacalc is an instance maintained by the organization framasoft.org (I am not in the dev team, I just asked the team and forward the answer).

The repo is public on https://framagit.org/framasoft/framacalc and the script https://framagit.org/framasoft/framacalc/blob/master/static/framacalc.js could be of some help (because the link to a non-existent spreadsheet creates it, this tool just creates a new page name for the user). Because framasoft is a french organization, you could find some french comments but it should be understandable.

Is this what you were looking for ?
Hope this will help.

**Comment by mathieu-aubin** [comment]
Thank you.
Honestly, there's nothing complicated in doing all of this. I just don't have time for the moment. I have a bunch of projects i am working on currently... but i will get to do this don't you worry.
